### ISSUE TITLE
CopyQ do not paste on Mac OS Mojave 10.14.3

### ISSUE THREAD

**Issue by ottopic** [hluk/CopyQ#1091]
Last updated version do not paste selected clip. I tried on to Mac with the same Mac OS version.
Rollback to previous version all works fine.

**Comment by Fitech** [comment]
Yes, I can confirm this bug.

**Comment by CrajyB** [comment]
@hluk I'm guessing you have not been able to reproduce this issue. How can we help you figure this one out? If you can provide any guidance, I'd be more than happy to assist with getting you what you need to troubleshoot this.

**Comment by hluk** [comment]
@CrajyB I tested this on older Mac. I guess this is issue on Mojave only.

Figuring out version or even commit which broke it might help but I don't think I'll be able to test it myself.

**Comment by CrajyB** [comment]
@hluk yep as I was thinking. Maybe its about time I get this project setup and do some actual development on it. I'll search the repo for dev documentation to setup a local env so I can help. If I could walk through the code live, maybe that would help.

**Comment by hluk** [comment]
@CrajyB If you could also help with this, that would be awesome!

BTW, the code related to this is in [src/platform/mac/macplatformwindow.mm](https://github.com/hluk/CopyQ/blob/a5e1e9f935df924ecc250d9760610507c6aafce4/src/platform/mac/macplatformwindow.mm).

I don't see any recent changes that could break the behavior so maybe the code just needs to be updated to work with Mojave.

I think some of the platform-specific code still uses a deprecated API (compiler warnings shows up).

**Comment by ottopic** [comment]
I try with new 3.8.0 version. After replacing old copyQ app, paste doesn't works anymore. I tried to reboot Mac, trying to using paste again finally osx ask me to enable accessibility permissions. With permissions enabled all works fine. 
I think this shouldn't happen, permissions should be requested at first run of updated version not after reboot.

**Comment by jimpriest** [comment]
Ran into this myself - had it in my permissions but it wasn't working.  Removed it and re-added it back and it works.  Maybe needs to be 'reset' after an update?

**Comment by hluk** [comment]
[This is already described in FAQ](https://copyq.readthedocs.io/en/latest/faq.html#why-can-i-no-longer-paste-from-the-application-on-macos).

Maybe something needs to be added to [Info.plist](https://github.com/hluk/CopyQ/blob/master/shared/Info.plist) to avoid doing this multiple times.
